src/plugins/dashboard/public/application/listing/__snapshots__/dashboard_listing.test.js.snap

```diff
@@ -56,7 +65,7 @@ exports[`after fetch hideWriteControls 1`] = `
         },
       ]
     }
-    tableListTitle="Dashboards"
+    tableListTitle="Overview"
```

Comment:
The screenshot in the PR does not have this change. Did you get UX signoff for this?

Reply:
Yes it is signed off by @kgcreative